rules/terraformrules/terraform_comment_syntax.go

```diff
@@ -38,7 +38,7 @@ func (r *TerraformCommentSyntaxRule) Link() string {
 	return tflint.ReferenceLink(r.Name())
 }
 
-// Check checks whether variables have type
+//  checks whether single line comments is used
```

Comment:
golint enforces method comments to start with that method name.

```suggestion
//  Check checks whether single line comments is used
```

Reply:
Thanks Kazuma for letting me know. I was able to complete the build now after updating the comment again